simulariumio/springsalad/springsalad_converter.py

```diff
@@ -90,11 +114,11 @@ def _parse_springsalad_data(
                     if raw_type_name in input_data.display_data
                     else raw_type_name
                 )
-                result.positions[time_index][
-                    agent_index
-                ] = input_data.meta_data.scale_factor * np.array(
+                position = input_data.meta_data.scale_factor * np.array(
                     [float(cols[4]), float(cols[5]), float(cols[6])]
                 )
+                scene_agent_positions[int(cols[1])] = position
+                result.positions[time_index][agent_index] = position
```

Comment:
Sorry... what is the difference between `scene_agent_positions` and `result.positions`?